public/static/docs/command-reference/gc.md

```diff
@@ -14,9 +15,13 @@ usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r REMOTE] [-f] [-j JOBS]
 This command deletes (garbage collects) data files or directories that may exist
 in the cache (or [remote storage](/doc/command-reference/remote) if `-c` is
 used) but no longer referenced in [DVC-files](/doc/user-guide/dvc-file-format)
-currently in the <abbr>workspace</abbr>. By default, this command only cleans up
-the local cache, which is typically located on the same machine as the project
-in question. This usually helps to free up disk space.
+currently in the <abbr>workspace</abbr>. By default, this command won't delete
+anything at all to make it safe and explicit. However, you can use different
```

Comment:
> By default, this command won't delete anything at all.

Is this true? `dvc gc` doesn't do anything? Does it output a warning about this at least?

Reply:
Yes, it doesn't *do* anything. And, it does show error message if tried without any flags. Though for the docs, this is not important.

Reply:
Thanks for the info. I'm not so sure this is unimportant for docs but will follow up in https://github.com/iterative/dvc.org/pull/1023#discussion_r386595996 below. Resolving 🙂